### ISSUE TITLE
Adjust timestamp to have a proper timezone

### ISSUE THREAD

**Issue by mikhail-zlochevskyi** [DeFiCh/metascan#253]
#### What happened:
Adjust timestamp to the format provided in blockscout. 

<img width="1378" alt="image" src="https://github.com/BirthdayResearch/metascan/assets/2522289/ab6bfbdb-71c4-4032-89d5-d1f06bd3f48c">

#### What you expected to happen:

<img width="882" alt="image" src="https://github.com/BirthdayResearch/metascan/assets/2522289/c601c2a3-93f0-4e54-8cfc-67dc3aa47e74">

**Comment by bryan-seto** [comment]
Makes sense that it looks slightly confusing even when other scanners are using the similar format. KIV and parked under debt first, we can return to this when the must-have and should-have features are shipped
